### ISSUE TITLE
bug: Frontend forms fail to load numbers with the value 0, or enter a negative number

### ISSUE THREAD

**Issue by ogenstad** [opsmill/infrahub#3640]
### Component

Frontend UI

### Current Behavior

When you create an object using the frontend it's not possible to enter a negative number, also when you enter a 0 the "0" is displayed to the right side of the value input field.

While it's possible to save a node with a 0 value for the number when you try to edit the object the 0 is removed so it looks like the value isn't set.

### Expected Behavior

It should be possible to edit nodes with a value of 0 and create and edit nodes with a negative value for a Number kind. When entering 0, this value should not be displayed beside the input box.

### Steps to Reproduce

1. Load this schema in Infrahub:

```yaml
# yaml-language-server: $schema=https://schema.infrahub.app/develop/schema.schema.json
---
version: '1.0'
nodes:
  - name: Number
    namespace: Test
    label: "number"
    display_labels:
      - primary__value
    attributes:
      - name: primary
        kind: Number
        optional: false
      - name: secondary
        kind: Number
        optional: true
```

2. Try to create an object of this kind, first note that it's not possible to enter a negative value
3. Enter 0 as the value for both fields
4. Edit the object you created and note that the values that should have had 0 as the previous value doesn't have a value at all.

**Comment by bilalabbad** [comment]
PR open to fix when default value is `0`.

But I could not reproduce the negative value issue. Do you still encounters it?

**Comment by ogenstad** [comment]
Thanks the issue with `0` got fixed in #3678, and the new forms probably fixed the issue with the negative numbers.
